ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/openshift-e2e-ibmcloud-heterogeneous-upi-workflow.yaml

```diff
@@ -6,8 +6,10 @@ workflow:
     - ref: ipi-conf-etcd-on-ramfs
     - chain: ipi-install
     - ref: upi-install-heterogeneous-powervs
+    - ref: enable-qe-catalogsource
```

Comment:
Let's create a new workflow? maybe call it cucushift? ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/cucushift? 

it may also need to go to: 

ci-operator/step-registry/cucushift/installer/rehearse/powervs/ see git/openshift/release/ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/cucushift-installer-rehearse-gcp-upi-workflow.yaml as an example

We probably don't want to modify the existing workflow.

/